Translators/WZDx/main.py

```diff
@@ -34,6 +35,14 @@ def translate(wzdx_geojson):
     for feature in wzdx_geojson["features"]:
         tim_body = generate_tim(feature)
         if tim_body is not None:
+            for msg in tim_body["dataframes"]:
+                # change 'workzone' to 'workZone' before appending TIM
+                if msg["content"] == "workzone":

```

Comment:
Should we just do this in the `tim_generator.py` in the first place? We're setting it to the lowercase in there currently, we could just set it properly the first round.

Reply:
Changed tim_generator to use workZone instead of workzone, removed this from main